static/app/views/explore/charts/widgetExtrapolationFooter.tsx

```diff
@@ -1,27 +1,18 @@
 import styled from '@emotion/styled';
 
-import Count from 'sentry/components/count';
-import {SegmentedLoadingBar} from 'sentry/components/segmentedLoadingBar';
```

Comment:
@Zylphrex I just noticed that `SegementedLoadingBar` is now completely unused apart from its own `stories` file. Can you remove that one as well please?